Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingBottomSheetViewController.swift

```diff
@@ -0,0 +1,220 @@
+//
+// Wire
+// Copyright (C) 2022 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+import WireDataModel
+import WireSyncEngine
+import avs
+
+protocol CallInfoConfigurationObserver: AnyObject {
+    func didUpdateConfiguration(configuration: CallInfoConfiguration)
+}
+
+class CallingBottomSheetViewController: BottomSheetContainerViewController {
+    private let bottomSheetMaxHeight = UIScreen.main.bounds.height * 0.7
+
+    weak var delegate: ActiveCallViewControllerDelegate?
+    private var participantsObserverToken: Any?
+    private let voiceChannel: VoiceChannel
+    private let headerBar = CallHeaderBar()
+    private let overlay = PassThroughOpaqueView()
+
+    var bottomSheetMinimalOffset: CGFloat {
+        switch voiceChannel.state {
+        case .incoming:
+            return 230.0
+        default:
+            return 128.0
+        }
+    }
+
+    let callingActionsInfoViewController: CallingActionsInfoViewController
+    var visibleVoiceChannelViewController: CallViewController {
+        didSet {
+            transition(to: visibleVoiceChannelViewController, from: oldValue)
+        }
+    }
+
+    init(voiceChannel: VoiceChannel) {
+        self.voiceChannel = voiceChannel
+        let selfUser: UserType = ZMUser.selfUser()
+        visibleVoiceChannelViewController = CallViewController(voiceChannel: voiceChannel, selfUser: selfUser, isOverlayEnabled: false)
+        callingActionsInfoViewController = CallingActionsInfoViewController(participants: voiceChannel.getParticipantsList(), selfUser: selfUser)
+        super.init(contentViewController: visibleVoiceChannelViewController, bottomSheetViewController: callingActionsInfoViewController, bottomSheetConfiguration: .init(height: bottomSheetMaxHeight, initialOffset: 128.0))
```

Comment:
This looks like quite long. Let's try to make it shorter:
```suggestion
        super.init(contentViewController: visibleVoiceChannelViewController, 
                       bottomSheetViewController: callingActionsInfoViewController, 
                       bottomSheetConfiguration: .init(height: bottomSheetMaxHeight, 
                                                                              initialOffset: 128.0))
```